test/generic/Ideal-test.jl

```diff
@@ -304,6 +304,60 @@ end
    end
 end
 
+
+@testset "Generic.Ideal.membership" begin
+   # multivariate
+   R, (x, y) = polynomial_ring(ZZ, ["x", "y"]; internal_ordering=:degrevlex)
+
+   ex = example_ideal_gens(x, y)
+   for V in ex[1:15], W in ex[1:15]
+
+      I = Generic.Ideal(R, V)
+
+      @test all(x in I for x in V)
```

Comment:
```suggestion
      @test all(in(I), V)
```
this is possible to do without an anonymous function. but no strong preference